ui/src/app/components/copy-modal.tsx

```diff
@@ -333,16 +354,16 @@ class CopyModalComponent extends React.Component<Props, State> {
           onChange={(destWorkspace) => this.validateAndSetDestination(destWorkspace)}
           isOptionDisabled={(option) => this.isDestinationSameWorkspace(option.value)}
         />
-        {cdrMismatch && resourceType === ResourceType.CONCEPTSET && <ConceptSetCdrMismatch text={cdrMismatch}/>}
+        {accessTierMismatch && <AccessTierMismatch text={accessTierMismatch}/>}
+        {!accessTierMismatch && cdrMismatch && resourceType === ResourceType.CONCEPTSET && <ConceptSetCdrMismatch text={cdrMismatch}/>}
```

Comment:
nit: this (line number 358 and 366) condition can be moved to a function, which can accept a ResourceType, eg
function (rt) {
return !accessTierMismatch && cdrMismatch && resourceType === rt;
}

Reply:
sgtm